CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ For customers using the new August 2023 schema, please note the following change
 - Added the fields `message_type_created_at, frequency_cap, rate_limit_per_minute, subscription_policy, and message_type_updated_at` to `stg_iterable__message_type`
 - Added the field `iterable_user_id` to `stg_iterable__user_history`
 
+## Bug Fixes
+- Explicitly cast the `email_list_ids` field in the `stg_iterable__user_history` model to be a string. This ensures the JSON as string operations in the downstream models are not impacted by errors if the connector syncs this field as a JSON datatype. ([PR #29](https://github.com/fivetran/dbt_iterable_source/pull/29))
```

Comment:
We can change the PR reference to fit however you are planning to use them in your PR.

Reply:
sounds good, will just merge in as is now and move some things around. I had added explicit casts for the other id fields, will make sure I just do it for all of them and note this in the changelog

Reply:
Merge conflicts, modified the changelog to account for all explicit casts being added.